source/math/FloatPoint2D.ooc

```diff
@@ -32,6 +32,10 @@ FloatPoint2D: cover {
 		(this scalarProduct(other) / (this norm * other norm)) clamp(-1, 1) acos() * (this x * other y - this y * other x < 0 ? -1 : 1)
 	}
 	distance: func (other: This) -> Float { (this - other) norm }
+	distanceSquared: func (other: This) -> Float {
+		point := this - other
```

Comment:
This name bothers me, but I'm not sure what to call it. The subtraction of a point with another is really a vector. Maybe `distanceVector`? What do you think?